README.md

```diff
@@ -139,6 +139,39 @@ dataset = TCRGraphDataset(
         )
 ```
 
+### To calculate TCR prediction metrics such as RMSD, interface RMSD (of the TCR:pMHC interface) or DockQ scores:
+To calculate DockQ scores first install DockQ: 
+```
+pip install DockQ
```

Comment:
Can this be added to the setup.py as a dependency to save having to install it separately?